File: examples/src/Text/ParagraphBullets.cs
using System.IO

import aspose.slides as slides
using System
import aspose.pydrawing as drawing
using Aspose.slides.Export

namespace Aspose.slides.Examples.CSharp.text
{
    public class ParagraphBullets
    {
        public static void Run()
        {
            #ExStart:ParagraphBullets
            # The path to the documents directory.
            dataDir = RunExamples.GetDataDir_Text()

            # Create directory if it is not already present.
            bool IsExists = System.IO.Directory.Exists(dataDir)
            if (!IsExists)
                System.IO.Directory.CreateDirectory(dataDir)

            # Creating a presenation instance
            with slides.Presentation() as pres:
            {

                # Accessing the first slide
                slide = pres.slides[0]


                # Adding and accessing Autoshape
                aShp = slide.shapes.add_auto_shape(ShapeType.Rectangle, 200, 200, 400, 200)

                # Accessing the text frame of created autoshape
                ITextFrame txtFrm = aShp.text_frame

                # Removing the default exisiting paragraph
                txtFrm.Paragraphs.remove_at(0)

                # Creating a paragraph
                Paragraph para = new Paragraph()

                # Setting paragraph bullet style and symbol
                para.ParagraphFormat.Bullet.type = BulletType.Symbol
                para.ParagraphFormat.Bullet.Char = Convert.ToChar(8226)

                # Setting paragraph text
                para.text = "Welcome to Aspose.Slides"

                # Setting bullet indent
                para.ParagraphFormat.Indent = 25

                # Setting bullet color
                para.ParagraphFormat.Bullet.Color.ColorType = ColorType.RGB
                para.ParagraphFormat.Bullet.Color.color = Color.Black
                para.ParagraphFormat.Bullet.IsBulletHardColor = 1 # set IsBulletHardColor to True to use own bullet color

                # Setting Bullet Height
                para.ParagraphFormat.Bullet.height = 100

                # Adding Paragraph to text frame
                txtFrm.Paragraphs.add(para)

                # Creating second paragraph
                Paragraph para2 = new Paragraph()

                # Setting paragraph bullet type and style
                para2.ParagraphFormat.Bullet.type = BulletType.Numbered
                para2.ParagraphFormat.Bullet.NumberedBulletStyle = NumberedBulletStyle.BulletCircleNumWDBlackPlain

                # Adding paragraph text
                para2.text = "This is numbered bullet"

                # Setting bullet indent
                para2.ParagraphFormat.Indent = 25

                para2.ParagraphFormat.Bullet.Color.ColorType = ColorType.RGB
                para2.ParagraphFormat.Bullet.Color.color = Color.Black
                para2.ParagraphFormat.Bullet.IsBulletHardColor = 1 # set IsBulletHardColor to True to use own bullet color

                # Setting Bullet Height
                para2.ParagraphFormat.Bullet.height = 100

                # Adding Paragraph to text frame
                txtFrm.Paragraphs.add(para2)


                #Writing the presentation as a PPTX file
                pres.save(dataDir + "Bullet_out.pptx", slides.export.SaveFormat.PPTX)

            }
            #ExEnd:ParagraphBullets
        }
    }
}

File: examples/src/Text/paragraph_bullets.py
import aspose.pydrawing as drawing
import aspose.slides as slides


def paragraph_bullets(global_opts):
    # Creating a presentation instance
    with slides.Presentation() as pres:
        # Accessing the first slide
        slide = pres.slides[0]

        # Adding and accessing Autoshape
        auto_shape = slide.shapes.add_auto_shape(slides.ShapeType.RECTANGLE, 200, 200, 400, 200)

        # Accessing the text frame of created autoshape
        text_frame = auto_shape.text_frame

        # Removing the default existing paragraph
        text_frame.paragraphs.remove_at(0)

        # Creating a paragraph
        para = slides.Paragraph()

        # Setting paragraph bullet style and symbol
        para.paragraph_format.bullet.type = slides.BulletType.SYMBOL
        para.paragraph_format.bullet.char = chr(8226)

        # Setting paragraph text
        para.text = "Welcome to Aspose.Slides"

        # Setting bullet indent
        para.paragraph_format.indent = 25

        # Setting bullet color
        para.paragraph_format.bullet.color.color_type = slides.ColorType.RGB
        para.paragraph_format.bullet.color.color = drawing.Color.black
        # set is_bullet_hard_color to True to use own bullet color
        para.paragraph_format.bullet.is_bullet_hard_color = slides.NullableBool.TRUE

        # Setting Bullet Height
        para.paragraph_format.bullet.height = 100

        # Adding Paragraph to text frame
        text_frame.paragraphs.add(para)

        # Creating second paragraph
        para2 = slides.Paragraph()

        # Setting paragraph bullet type and style
        para2.paragraph_format.bullet.type = slides.BulletType.NUMBERED
        para2.paragraph_format.bullet.numbered_bullet_style = slides.NumberedBulletStyle.BULLET_CIRCLE_NUM_WD_BLACK_PLAIN

        # Adding paragraph text
        para2.text = "This is numbered bullet"

        # Setting bullet indent
        para2.paragraph_format.indent = 25

        para2.paragraph_format.bullet.color.color_type = slides.ColorType.RGB
        para2.paragraph_format.bullet.color.color = drawing.Color.black
        # set is_bullet_hard_color to True to use own bullet color
        para2.paragraph_format.bullet.is_bullet_hard_color = slides.NullableBool.TRUE

        # Setting Bullet Height
        para2.paragraph_format.bullet.height = 100

        # Adding Paragraph to text frame
        text_frame.paragraphs.add(para2)

        # Writing the presentation as a PPTX file
        pres.save(global_opts.out_dir + "text_paragraph_bullets_out.pptx", slides.export.SaveFormat.PPTX)
